src/DependencyInjection/Configuration.php

```diff
@@ -38,7 +38,14 @@ public function getConfigTreeBuilder()
                         ->end()
                     ->end()
                 ->end()
-
+                ->arrayNode('video_media_queries')
```

Comment:
Diese Konfiguration müsste noch angepasst werden: 
Zum einen würde ich den Punkt nur "media_queries" nennen, wir sind ja schon im Video-Kontext (da bin ich aber offen für Diskussionen ;) )
Zum anderen wollten wir den Namen bearbeitbar halten, das bedeutet, der Name muss änderbar sein, ohne dass man alle Einstellungen anpassen muss. Ich dachte da an:
`'alias' => ['name' => 'Name', 'query' => 'Query']`
Also im Prinzip müsstest du noch ein Child namens "name" einführen und den Key in Alias umbenennen. Wäre mein Vorschlag. Sorry, dass ich das vorher übersehen habe.

Reply:
Der Name ist bearbeitbar, ein Beispiel:
video_media_queries:
    xs:
      query: 'screen and (min-device-width: 1px and max-device-width: 569px)'
    sm:
      query: 'screen and (min-device-width: 570px and max-device-width: 719px)'

das xs und sm kann alles sein was es will, hab zuerst auch einen Namen gemacht, so wie du es vorschlägst, es war aber unnötig bei der Konfiguration, es wäre dann folgende Konfiguration:

video_media_queries:
    xs:
      name: 'xs'
      query: 'screen and (min-device-width: 1px and max-device-width: 569px)'
    sm:
      name: 'sm'
      query: 'screen and (min-device-width: 570px and max-device-width: 719px)'

Reply:
Wie bereits besprochen, sollten wir name anbieten. Damit man xs später in "mobile ansicht" umbenennen könnte, ohne bestehende Konfigurationen zu zerstören.

Reply:
Passt so, denke ich!